packages/config/config/devices/0x031e/lzw60.json

```diff
@@ -30,19 +32,62 @@
 			"allowManualEntry": true
 		},
 		"12": {
-			"label": "PIR Sensitivity",
-			"description": "Change the sensitivity of the PIR (motion) sensor; 0 = off, 1 (low) - 10 (high)",
+			"label": "Motion Sensor Sensitivity",
+			"description": "Sensitivity level of the motion sensor. 0=Disabled 1=Low 10=High",
 			"valueSize": 1,
 			"minValue": 0,
 			"maxValue": 10,
 			"defaultValue": 8,
 			"unsigned": true,
 			"readOnly": false,
 			"writeOnly": false,
-			"allowManualEntry": true
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "1 - Low",
+					"value": 1
+				},
+				{
+					"label": "2",
+					"value": 2
+				},
+				{
+					"label": "3",
+					"value": 3
+				},
+				{
+					"label": "4",
+					"value": 4
+				},
+				{
+					"label": "5",
+					"value": 5
+				},
+				{
+					"label": "6",
+					"value": 6
+				},
+				{
+					"label": "7",
+					"value": 7
+				},
+				{
+					"label": "8",
+					"value": 8
+				},
+				{
+					"label": "9",
+					"value": 9
+				},
+				{
+					"label": "10 - High",
+					"value": 10
+				}
+			]
 		},
```

Comment:
I'd leave this on `allowManualEntry: true` and add a single option for disabled (0)